File: BEE 1012.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        float a = input.nextFloat();
        float b = input.nextFloat();
        float c = input.nextFloat();
        System.out.printf("TRIANGULO: %.3f\n", ((1.0/2)*a*c));
        System.out.printf("CIRCULO: %.3f\n", (3.14159*(c*c)));
        System.out.printf("TRAPEZIO: %.3f\n", ((1.0/2)*(a+b)*c));
        System.out.printf("QUADRADO: %.3f\n", (b*b));
        System.out.printf("RETANGULO: %.3f\n", (a*b));
    }
}

File: BEE 1012.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var line = input.split(' ');

a = parseFloat(line.shift());
b = parseFloat(line.shift());
c = parseFloat(line.shift());

console.log("TRIANGULO:", ((1.0/2)*a*c).toFixed(3));
console.log("CIRCULO:", (3.14159*(c*c)).toFixed(3));
console.log("TRAPEZIO:", ((1.0/2)*(a+b)*c).toFixed(3));
console.log("QUADRADO:", (b*b).toFixed(3));
console.log("RETANGULO:", (a*b).toFixed(3));